openmetadata-service/src/main/java/org/openmetadata/service/resources/teams/UserResource.java

```diff
@@ -1040,6 +1055,101 @@ public Response refreshToken(
     return Response.status(Response.Status.OK).entity(authHandler.getNewAccessToken(refreshRequest)).build();
   }
 
+  @GET
+  @Path("/token/personal")
```

Comment:
should we prefix /security/token under user, we can move our /token/

Reply:
prefixed with /security